1_crosswalk_fetch/src/fetch_crosswalk.R

```diff
@@ -32,6 +32,54 @@ process_wbic_lakes <- function(file_out = '1_crosswalk_fetch/out/wbic_lakes_sf.r
   saveRDS(wbic_lakes, file = file_out)
 }
 
+fetch_isro_lakes <- function(out_ind, zip_ind, remove_IDs = c(), gdb_filename, use_geoms){
+  outfile <- as_data_file(out_ind)
+
+  zip_file <- sc_retrieve(zip_ind)
+
+  gdb_path <- tempdir()
+  unzip(zip_file, exdir = gdb_path)
+
+
+  st_layer_info <- st_layers(file.path(gdb_path, gdb_filename))
+  layer_info <- tibble(layer_name = st_layer_info$name,
+                       geom_type = unlist(st_layer_info$geomtype),
+                       n_feature = st_layer_info$features,
+                       n_col = st_layer_info$fields)
+
+  # could replace this comment with sprintf for "use_geoms"
+  message('...dropping any layer that is not Multi Polygon, likely more than five lost...')
+
+  use_layers <- layer_info %>% filter(!is.na(geom_type) & n_feature > 1 & geom_type == use_geoms) %>%
+  # all but one remaining have 8 cols (one has 9)
+  # depth field is "Avg_Depth" for many even though that name is misleading, seems an int
+  # or "Depth_calc" when 9 cols
+    mutate(depth_col = case_when(
+      n_col == 9 ~ "Depth_calc",
+      n_col == 8 ~ "Avg_Depth"
+    )) %>%
+    dplyr::select(layer_name, depth_col)
+
+  data_clean <- purrr::pmap(use_layers, function(layer_name, depth_col){
+    # get "Beaver" from "Beaver_CntrPolys" etc:
+    lake_name <- strsplit(layer_name, '_')[[1]][1]
+
+    st_read(file.path(gdb_path, gdb_filename), layer_name, quiet = TRUE) %>%
+      sf::st_zm() %>% st_transform(crs = 4326) %>%
+      rename(geometry = Shape) %>%
+      mutate(site_id = sprintf('isro_%s', lake_name)) %>%
```

Comment:
This is lowercase but later in `2_crosswalk_munge` you pass in `ISRO_` to `poly1_ID_name`. Does that matter?

Reply:
I don't think we're consistent on this in the repo...but the reason for this is that `isro_{asdfasdf}` is the ISRO_ID field value, while the `ISRO_ID` that appears as an input value for `poly1_ID_name` in the `crosswalk_poly_intersect_poly()` function is the name of the field/column. I think for the most part the field/column name for these other IDs are all in caps, and for the most part (but I'm sure there are exceptions) the _value_ in that column is prefixed with the lowercase of it.

Reply:
Hmmm ok, I think [the value for the UNDERC is capitalized](https://github.com/USGS-R/lake-temperature-model-prep/pull/327/files#diff-4ae5528d8c4172ac5cc98d0b5259d450cf4be0ba079243f68e54dc33a295c859R311). Sounds like this isn't a hidden bug or anything, just more of a style issue.